src/release-pr.ts

```diff
@@ -111,11 +98,13 @@ export class ReleasePR {
     this.repoUrl = options.repoUrl;
     this.token = options.token;
     this.path = options.path;
-    this.packageName = options.packageName;
+    // If no packageName has been provided, use the name
+    // of the repository:
+    const [, repo] = parseGithubRepoUrl(this.repoUrl);
+    this.packageName = options.packageName || repo;
```

Comment:
huh, so it looks like `github-release` effectively [requires a package name](https://github.com/googleapis/release-please/blob/ce0d2d8d5bc1b278b5a5f8ebd81c6ba16db60645/src/github-release.ts#L80). `release-pr` does not... but if we want to set one by default here I think we should follow that same lookup logic rather than just choosing the repo name?

For consistency I'd actually like to see `release-pr` require it too and remove the concept of choosing wether to `OpenPROptions.includePackageName` But that would be a breaking change (breaking how last release pr is found) and maybe the asymmetry between including the package name in the release/tag but not having it show up in the PR is fine... probably just my AR/OC tendencies getting tickled :-)

Reply:
I was on the fence about this change too, to minimize the code change in this PR perhaps I should just default to an empty string -- and we can improve things later?

Reply:
@joeldodge79 I suggest we just start with the empty string.

Reply:
Some release strategies prefer not having a package name in the release title (for Java, we usually just have the release title be `v1.2.3` rather than `some-library-name v1.2.3`.